internal/webhook/external/github.go

```diff
@@ -164,7 +165,24 @@ func (g *githubWebhookReceiver) getHandler(requestBody []byte) http.HandlerFunc
 				xhttp.WriteErrorJSON(w, err)
 				return
 			}
-			repoURL = image.NormalizeURL(ref.Context().Name())
+			manifest := pkg.GetPackageVersion().GetContainerMetadata().GetManifest()
+			// Determine if the package was a Helm chart
+			if cfgAny, ok := manifest["config"]; ok {
+				if cfg, ok := cfgAny.(map[string]any); ok {
+					if mediaTypeAny, ok := cfg["media_type"]; ok {
+						if mediaType, ok := mediaTypeAny.(string); ok {
+
+							if mediaType == helmChartMediaType {
+								repoURL = helm.NormalizeChartRepositoryURL(ref.Context().Name())
+							}
+						}
+					}
+				}
+			}
```

Comment:
Might be slightly neater to do something like:

```go
func extractMediaType(manifest map[string]any) (string, bool) {
    config, ok := manifest["config"].(map[string]any)
    if !ok {
        return "", false
    }
    mediaType, ok := config["media_type"].(string)
    return mediaType, ok
}
```

Reply:
Ok. That's a neat trick. It didn't occur to be that I could do a retrieval and a type coercion in one fell swoop.

Fixed in 13b26915738f59117195f405213776d3f3a013bc